docs/src/reference/asciidoc/appendix/breaking.adoc

```diff
@@ -23,10 +23,32 @@ We've increased the `es.scroll.size` default from `50` to `1000`. If you deploy
 ES-hadoop in a low memory environment, consider lowering `es.scroll.size` to
 avoid issues.
 
-==== Support for Scala 2.10 has been removed
+==== Deprecations
+
+The following are deprecated and will be removed in a future release:
```

Comment:
I'd steal some of the wording from the [Elasticsearch migration guide here](https://www.elastic.co/guide/en/elasticsearch/reference/8.0/migrating-8.0.html#deprecated-8.0).

I'd also bump the heading up so that it's at the same level as the 8.0 breaking changes. That made the on-page TOC a little more readable to me.

This also adjusts the tags so they only include the changes/deprecations.

```suggestion
// end::notable-v8-breaking-changes[]

[[deprecations-8.0]]
=== Deprecations in 8.0

The following functionality has been deprecated in {eh} 8.0 and will be removed
in a future version. While this won’t have an immediate impact on your
applications, we strongly encourage you take the described steps to update your
code after upgrading to 8.0.

// tag::notable-v8-breaking-changes[]
```